Calculator/Calculator/CalculatorTest/FormulaTest/FormulaTest.swift

```diff
@@ -9,66 +9,66 @@ import XCTest
 @testable import Calculator
 
 final class FormulaTest: XCTestCase {
+    var sut: Formula!
     
     override func setUpWithError() throws {
         try super.setUpWithError()
+        sut = Formula(operands: CalculatorItemQueue(), operators: CalculatorItemQueue())
     }
 
     override func tearDownWithError() throws {
         try super.tearDownWithError()
+        sut = nil
     }
 
     func test_2더하기3이_들어있을때_result실행시_결과는5가된다() {
-        var operands = CalculatorItemQueue<Double>()
-        var operators = CalculatorItemQueue<Operator>()
         //given
-        operands.enqueue(2.0)
-        operands.enqueue(3.0)
-        operators.enqueue(.add)
+        sut.operands.enqueue(2.0)
+        sut.operands.enqueue(3.0)
+        sut.operators.enqueue(.add)
         
         let expectation = 5.0
         //when
-        var formula = Formula(operands: operands, operators: operators)
-        let result = try? formula.result()
+        let result = try? sut.result()
         //then
         XCTAssertEqual(result, expectation)
     }
 
     func test_2곱하기3과4가_들어있을때_result실행시_결과는6이된다() {
-        var operands = CalculatorItemQueue<Double>()
-        var operators = CalculatorItemQueue<Operator>()
         //given
-        operands.enqueue(2.0)
-        operands.enqueue(3.0)
-        operands.enqueue(4.0)
-        operators.enqueue(.multiply)
-        
+        sut.operands.enqueue(2.0)
+        sut.operands.enqueue(3.0)
+        sut.operands.enqueue(4.0)
+        sut.operators.enqueue(.multiply)
         
         let expectation = 6.0
         //when
-        var formula = Formula(operands: operands, operators: operators)
-        let result = try? formula.result()
+        let result = try? sut.result()
         //then
         XCTAssertEqual(result, expectation)
     }
     
     func test_2빼기3곱하기4가_들어있을때_result실행시_결과는마이너스4가된다() {
-        var operands = CalculatorItemQueue<Double>()
-        var operators = CalculatorItemQueue<Operator>()
         //given
-        operands.enqueue(2.0)
-        operands.enqueue(3.0)
-        operands.enqueue(4.0)
-        operators.enqueue(.substract)
-        operators.enqueue(.multiply)
-        
+        sut.operands.enqueue(2.0)
+        sut.operands.enqueue(3.0)
+        sut.operands.enqueue(4.0)
+        sut.operators.enqueue(.substract)
+        sut.operators.enqueue(.multiply)
         
         let expectation = -4.0
         //when
-        var formula = Formula(operands: operands, operators: operators)
-        let result = try? formula.result()
+        let result = try? sut.result()
         //then
         XCTAssertEqual(result, expectation)
     }
 
+    func test_1나누기0이_들어있을때_result실행시_divideByZero_error가난다() {
+        //given
+        sut.operands.enqueue(1)
+        sut.operands.enqueue(0)
+        sut.operators.enqueue(.divide)
+        // when, then
+        XCTAssertThrowsError(try sut.result()) { error in XCTAssertEqual(error as? CalculatorError, CalculatorError.divideByZero)}
```

Comment:
👍👍👍👍

후행 클로저에서 축약형을 사용하지 않고 파라미터를 명시해주는 경우에는
아래와 같이 개행을 추가해주는 편입니다.
```suggestion
        XCTAssertThrowsError(try sut.result()) { error in
            XCTAssertEqual(error as? CalculatorError, CalculatorError.divideByZero)
        }
```